### PR TITLE
capture and display cancelled unscheduled contacts

### PR DESCRIPTION
**Pull request by ericcf**
[#116310725]

### CODE REVIEW

File: app/models/help_request_call.rb [hunk 1]
```diff
@@ -1,7 +1,17 @@
...
 # A record of a call placed by a Nurse to a Participant.
 class HelpRequestCall < ActiveRecord::Base
+  CANCELLED = "CANCEL"
+
   belongs_to :participant
 
   validates :participant, :contact_at, :explanation, presence: true
+
+  def self.cancelled
+    where(arel_table[:explanation].matches("#{CANCELLED}%"))
```
Comment (michaelwehrley) [inline app/models/help_request_call.rb]: why the `%`?

File: spec/views/nurse_supervisor_dashboards/show.html.erb_spec.rb [hunk 2]
```diff
@@ -274,5 +276,22 @@ def assign_nurses_and_render
       expect(rendered).to include "Unit 1"
       expect(rendered).to include "Unit 2"
     end
+
+    it "renders the reasons for cancelled unscheduled contacts" do
```
Comment (michaelwehrley) [inline spec/views/nurse_supervisor_dashboards/show.html.erb_spec.rb]: Nice spec.

File: app/models/help_request_call.rb (hunk 1 above)
Comment (ericcf) [inline app/models/help_request_call.rb]: This creates an SQL `LIKE` query, where `%` is a wildcard (like `*` in a regular expression). See http://www.postgresql.org/docs/9.3/static/functions-matching.html.

**Comment by michaelwehrley** [comment]
Looks great to me.  I just don't understand the `%`.  I thought it was a regex matcher but that doesn't make sense, and I don't see it listed in the arel documents.  Good work!

File: app/models/help_request_call.rb (hunk 1 above)
Comment (michaelwehrley) [inline app/models/help_request_call.rb]: Gotcha.  Thanks!

**Pull request merged**